core_lib/src/selectionpainter.cpp

```diff
@@ -5,6 +5,8 @@
 #include "layermanager.h"
```

Comment:
Managers should not exist inside painter methods, the paint method should only take in parameters requires.